service/gcs/main.go

```diff
@@ -15,21 +18,39 @@ import (
 )
 
 func main() {
-	// parse command line parameters and init logger
-	if err := utils.ProcessCommandlineOptions(); err != nil {
-		logrus.Fatalf("%+v", err)
+	logLevel := flag.String("loglevel", "warning", "Logging Level: debug, info, warning, error.")
+	logFile := flag.String("logFile", "stdout", "Logging Target: A file name/path or 'stdout' for console output.")
+
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "\nUsage of %s:\n", os.Args[0])
+		flag.PrintDefaults()
+		fmt.Fprintf(os.Stderr, "Examples:\n")
+		fmt.Fprintf(os.Stderr, "    %s -loglevel=verbose -logfile=gcslog.txt (default)\n", os.Args[0])
+		fmt.Fprintf(os.Stderr, "    %s -loglevel=verbose -logfile=stdout\n", os.Args[0])
+		fmt.Fprintf(os.Stderr, "    %s -loglevel=none \n", os.Args[0])
+	}
+
+	flag.Parse()
+
+	// Use a file instead of stdout
+	if !strings.EqualFold(*logFile, "stdout") {
```

Comment:
It's a little more unixy to make empty string mean stdout instead of a special token like this.